apps/browser/src/scripts.template.js

```diff
@@ -0,0 +1,72 @@
+const Handlebars = require("handlebars");
```

Comment:
This file should probably go within `browser/scripts/...` and get called within a prebuild script